discovery-provider/plugins/notifications/src/processNotifications/mappers/reaction.ts

```diff
@@ -109,13 +117,26 @@ export class Reaction extends BaseNotification<ReactionNotificationRow> {
       )
       await this.incrementBadgeCount(this.senderUserId)
     }
+
     if (
+      isLiveEmailEnabled &&
       userNotificationSettings.shouldSendEmail({
         initiatorUserId: this.receiverUserId,
         receiverUserId: this.senderUserId
       })
     ) {
-      // TODO: Send out email
+      const notification: AppEmailNotification = {
+        receiver_user_id: this.senderUserId,
+        ...this.notification
+      }
+      await sendNotificationEmail({
+        userId: this.receiverUserId,
```

Comment:
think we need to update `receiverUserId` here and the line below to be `senderUserId`